suricata.yaml.in

```diff
@@ -104,7 +104,7 @@ outputs:
       # Specify the amount of buffering, in bytes, for
       # this output type. The default value 0 means "no
       # buffering".
-      buffer-size: 0
+      #buffer-size: 0
```

Comment:
This is a bit confusing for me as usually in our config 0 means "get everything"/indefinite. Here, 0 actually means no buffering.

Reply:
Good point; a difference here is that this is a size value that expresses how much data should be buffered with the default being "no buffering".